### PR TITLE
move implementations from ethereum-ibc-relay-chain

### PR DESCRIPTION
**Pull request by dai1975**

### CODE REVIEW

File: proto/buf.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/datachainlab/ibc-hd-signer
+deps:
+  - buf.build/cosmos/gogo-proto
+  - buf.build/cosmos/ibc
```
Comment (siburu) [inline proto/buf.yaml]: Maybe this repository only depends gogo-proto, and not ibc?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+DOCKER := $(shell which docker)
+
+protoVer=0.13.1
```
Comment (siburu) [inline Makefile]: Can we use `0.14.0`? Related projects (e.g. yui-relayer and ibc-go) use this version.

**Review by siburu — COMMENTED** [review]
@dai1975 Sorry for late reply. I put some comments.

File: proto/buf.yaml (hunk 1 above)
Comment (dai1975) [reply proto/buf.yaml]: fixed

File: Makefile (hunk 2 above)
Comment (dai1975) [reply Makefile]: fixed

**Pull request merged**